Playbooks/Create Incidents with Email/Readme.md

```diff
@@ -0,0 +1,46 @@
+
+
+
+
+Please use the below button to deploy
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FCreate%2520Incidents%2520with%2520Email%2Fazuredeploy.json)
+
+### Deployment
+
+Once deployed you can configure the connetions as below
+
+![image](https://user-images.githubusercontent.com/20562985/196198346-2dbde9ca-9812-4c49-a0e6-d3ebca777bb3.png)
+
+
+### Configuration
+
+1. Sentinel Connection
+
+![image](https://user-images.githubusercontent.com/20562985/196175586-0fd33803-6fd3-4429-8af4-945c8a0c8511.png)
+
+
+2. Office 365 Connection
+
```

Comment:
same here 

Screenshot is good , but it will look nicer if you can provide the step by step approach to create and configured the connection

Reply:
Same as above.